pysrc/whenever/__init__.pyi

```diff
@@ -1055,13 +1055,13 @@ class InvalidOffset(ValueError): ...
 class ImplicitlyIgnoringDST(TypeError): ...
 
 class Weekday(enum.Enum):
-    MONDAY = ...
-    TUESDAY = ...
-    WEDNESDAY = ...
-    THURSDAY = ...
-    FRIDAY = ...
-    SATURDAY = ...
-    SUNDAY = ...
+    MONDAY = int
```

Comment:
Ah--I mean literally put the values 1..7 here: `MONDAY = 1` etc.

Reply:
Ah oops

Reply:
No problem, type stubs can be unintuitive--that's the reason I didn't get it right the first time 🙂